service/windows/installer/Node.wxs

```diff
@@ -123,9 +123,6 @@
             <Component>
                 <File Name="liblzma-5.dll" Source="!(bindpath.lib)liblzma-5.dll" KeyPath="yes" />
             </Component>
-            <Component>
-                <File Name="libpq.dll" Source="!(bindpath.lib)libpq.dll" KeyPath="yes" />
```

Comment:
With the exception of `liblmdb.dll`, all of the other dlls can be removed as they are (recursive) dependencies of `libpq.dll`.

Reply:
I have removed them as requested.